src/blog/_posts/2023-06-08-survey-results.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 2023 Cadence Community Survey Results
+
+date: 2023-06-08
+author: Ender Demirkaya
+authorlink: https://www.linkedin.com/in/enderdemirkaya/
+---
+
+We released a user survey earlier this year to learn about who our users are, how they use Cadence, and how we can help them. After collecting the feedback, we wanted to share the results with our community. Thank you everyone for filling it out! Your feedback is invaluable and it helps us shape our roadmap for the future.
+
+
+Here are some highlights in text and you can check out the visuals to get more details:
+
+![using.png](../../shared/img/2023-survey-results/using.png)
+
+![job_role.png](../../shared/img/2023-survey-results/job_role.png)
+
+Most of the people who replied to our survey were engineers who were already using Cadence, actively evaluating, or migrating from a similar technology. This was exciting to hear! Some of you have contacted us to learn more about benchmarks, scale, and ideal use cases. We will share more guidelines about this but until then, feel free to contact us over our Slack workspace for guidance.
+
+![scale.png](../../shared/img/2023-survey-results/scale.png)
+
+The scale our users operating Cadence varies from thousands to billions of workflows per month. It was exciting to see it being used in both small and large scale companies.
+
+![time_zone.png](../../shared/img/2023-survey-results/time_zone.png)
+
+Our user base is the biggest in Europe compared to any other place. This is inline with Cadence team growing its presence in Europe. Users from different places also contacted us to contribute to Cadence as a follow up to the survey. We will start putting up-for-grabs tasks on Github. Several of them wanted to meet with a Zoom call and to discuss their use cases and best practices. As the Cadence team has presence in both the EU and the US, we welcome all our users to contact us anytime. Slack is the fastest way to reach us.
```

Comment:
Nit: `This is in-line with the Cadence team` 

`user-base` here refers to distinct respondees presumably, not customers, which would probably be the US?

Reply:
Up-for-grabs tasks, or `new-starter` tasks?